book/src/SUMMARY.md

```diff
@@ -3,6 +3,7 @@
 - [Introduction](introduction.md)
 - [Tutorial](tutorial/introduction.md)
   - [Creating a UEFI Application](tutorial/app.md)
+    - [Combining Rust `std` with `uefi`](tutorial/rust-std.md)
```

Comment:
I think this would be better under the how-to section rather than the tutorial. The tutorial should guide people to the recommended usage of the uefi crate, and right now we're not recommending using uefi-std.